content/news/playtest-20150102.md

```diff
@@ -0,0 +1,34 @@
+We wish everyone a happy new year and prepared a new playtest which builds upon our [Halloween release](/news/release-20141029/). The main focus was to improve stability and performance as well as implement unique features of Dune 2000 that distinguish it from the other mods.
+
+
+These playtest builds provide a snapshot of our current development for feedback by our testers. You can help shape the future of OpenRA by testing these builds and [reporting bugs](http://bugs.openra.net) or simply letting the [community](/community/) know what you think.
+
+Some highlights in this development build are:
+
+   - Added an often requested lobby option for disabling the "Short Game" mode.
+   - Added a difficulty selector for single-player missions and FMV briefings.
+   - Added hardware cursors support for smoother mouse movements.
+   - Added new missions: GDI 03, Soviet 01 and Survival 01.
+   - Fixed Fort Lonestar and the Dropzone minigames not starting.
+
+See the [changelog](http://changelog.openra.net) for more details on the playtest changes and bugfixes.
+
+The playtest-20150102 build is available from the [download](/download/) page. We handed over packaging in regards of RPM for Fedora and openSUSE as well as Arch Linux completely to the Linux community as they produce better results than our build servers. Note to packagers: we switched dependency management to [NuGet](http://www.nuget.org/).
+
+<div style="text-align:center" markdown="1">
+![Countries](/images/news/20150102-ra-countries.png)
+ 
+The Red Alert mod introduces support for sub-factions. These nations have special units or support powers, but otherwise have a lot in common with the Allied or Soviet army they belong to.
+
+![Game server browser](/images/news/20150102-cnc-colored-game-servers.png)
+
+We overhauled the server browser. It is grouped by mods with the currently loaded one on top, but you can now join any game if the mod is installed. The ingame-interface has been updated to use the same new color scheme as the web-based [game server overview](/games/). We also added a filter for password-protected servers.
+
+![Sandworms and Carryalls](/images/news/20150102-d2k-sandworm-carryall.png)
+ 
+Let your slow harvesters make use of the convenient airlift shuttle service by a fleet of carryalls and avoid your units becoming sandworm food! As always the unit balancing is not meant to be final. You can also turn off sandworms in the lobby options.
```

Comment:
This will read better if it is broken down into smaller sentences.  I suggest "Let your slow harvesters make use of the convenient airlift shuttle service, and save your units from becoming worm food!  Carryalls and Worms are now included in our D2K mod.  The worm balancing is a work in progress, and they can be disabled in the lobby options."